diff -up PyQt6-6.3.1/sip/QtGui/qevent.sip.1~ PyQt6-6.3.1/sip/QtGui/qevent.sip
--- PyQt6-6.3.1/sip/QtGui/qevent.sip.1~	2022-06-17 17:12:25.021261000 +0200
+++ PyQt6-6.3.1/sip/QtGui/qevent.sip	2022-09-18 21:41:06.513805833 +0200
@@ -293,6 +293,7 @@ class QCloseEvent : QEvent /NoDefaultCto
 public:
     QCloseEvent();
     virtual ~QCloseEvent();
+    virtual QCloseEvent *clone() const /Factory/;
 };
 
 class QIconDragEvent : QEvent /NoDefaultCtors/
@@ -304,6 +305,7 @@ class QIconDragEvent : QEvent /NoDefault
 public:
     QIconDragEvent();
     virtual ~QIconDragEvent();
+    virtual QIconDragEvent *clone() const /Factory/;
 };
 
 class QShowEvent : QEvent /NoDefaultCtors/
@@ -315,6 +317,7 @@ class QShowEvent : QEvent /NoDefaultCtor
 public:
     QShowEvent();
     virtual ~QShowEvent();
+    virtual QShowEvent *clone() const /Factory/;
 };
 
 class QHideEvent : QEvent /NoDefaultCtors/
@@ -326,6 +329,7 @@ class QHideEvent : QEvent /NoDefaultCtor
 public:
     QHideEvent();
     virtual ~QHideEvent();
+    virtual QHideEvent *clone() const /Factory/;
 };
 
 class QContextMenuEvent : QInputEvent /NoDefaultCtors/
@@ -464,6 +468,7 @@ class QDragEnterEvent : QDragMoveEvent /
 public:
     QDragEnterEvent(const QPoint &pos, Qt::DropActions actions, const QMimeData *data, Qt::MouseButtons buttons, Qt::KeyboardModifiers modifiers);
     virtual ~QDragEnterEvent();
+    virtual QDragEnterEvent *clone() const /Factory/;
 };
 
 class QDragLeaveEvent : QEvent /NoDefaultCtors/
@@ -475,6 +480,7 @@ class QDragLeaveEvent : QEvent /NoDefaul
 public:
     QDragLeaveEvent();
     virtual ~QDragLeaveEvent();
+    virtual QDragLeaveEvent *clone() const /Factory/;
 };
 
 class QHelpEvent : QEvent /NoDefaultCtors/
